For gcc 4.6 to build this. Otherwise it says 's unused variable', and it's built
with -Werror.

diff --git a/src/cert.c b/src/cert.c
index 28c7b11..34570a4 100644
--- a/src/cert.c
+++ b/src/cert.c
@@ -87,9 +87,7 @@ char *cp;
 	int i,j,t;
 
 	for(i=0;i<8;i++){
-		int s;
-                  
-                s = scanf("%2x",&t);
+                scanf("%2x",&t);
 		if(feof(stdin))
 		  good_bye();
 		for(j=0; j<8 ; j++) {
